setup.py

```diff
@@ -25,9 +25,13 @@ def readme():
           'pyhocon',
           'mock',
           'arrow',
-          'nose'
+          'nose',
+          'requests',
+          'thrift==1.0.0-py3',
+          'munch'
       ],
       entry_points={'console_scripts': ['herc = herc.webservice:main']},
+      dependency_links = [urllib.parse.urljoin('file:', os.path.join(os.getcwd(), 'thrift-1.0.0-py3.tar.gz'))],
```

Comment:
What's happening here is a version of [this](http://stackoverflow.com/a/3481388/2941784): specify a version that doesn't exist on PyPI, then satisfy that version yourself in dependency_links. The path joining is to use a local file as an install source, rather than specifying e.g. the GitHub tarball (which is more common).

I suspect this means the tarball should be added to `package_data` or `data_files` or whatever ends up being used (see above comments).